Document Clans, Member Verification, and Join Requests (#224)
discord-userdoccers/discord-userdoccers: pages/resources/user.mdx

  ]
}
```

<RouteHeader method="PUT" url="/users/@me/clan">

>   Set Clan

Reviewer: ```suggestion
  Set Clan Identity
```

</RouteHeader>

> Sets the current user's clan. Returns a partial [user](#user-object) object of before the update.

Reviewer: ```suggestion
Sets the current user's clan. Returns an outdated partial [user](#user-object) object on success.
```

###### JSON Params

| Field             | Type       | Description                                                       |
| ----------------- | ---------- | ----------------------------------------------------------------- |

> | identity_enabled  | boolean    | Whether or not the clan tag is enabled                            |

Reviewer: ```suggestion
| identity_enabled  | boolean    | Whether the clan tag is enabled                                  |
```

> | identity_guild_id | ?snowflake | The guild ID of the clan to be associated with, or empty to clear |

Reviewer: ```suggestion
| identity_guild_id | ?snowflake | The guild ID of the clan to be associated with, if any            |
```
